### ISSUE TITLE
Update package upload/release docs to include huber and arkade

### ISSUE THREAD

**Issue by gaurav-gogia** [tenable/terrascan#1428]
### Description
[Arkade](https://github.com/alexellis/arkade)
[Huber](https://github.com/innobead/huber)

**Comment by vedant-z** [comment]
@gaurav-gogia I would like to work on this.

**Comment by gaurav-gogia** [comment]
@vedant-z 

Three simple steps

1. Visit Arkade & Huber websites. 
2. Read about how new packages are uploaded there. 
3. Write the same explanation in simple words when you update Terrascan documentation.

**Comment by vedant-z** [comment]
Hi @gaurav-gogia if I am not wrong I need to explain these topic in simpler terms in Installation doc right?

Also could you please point out the specific doc intended for Installation, that would be of great help

**Comment by vedant-z** [comment]
Hi @gaurav-gogia if I am not wrong I need to explain these topic in simpler terms in Installation doc right?

Also could you please point out the specific doc intended for Installation, that would be of great help

**Comment by vedant-z** [comment]
@gaurav-gogia Can you please provide the relevant links of Arkade and Huber, as there are many similar sites creating some confusion.
